packages/claim-cli/src/applications/check-eligibility.ts

```diff
@@ -0,0 +1,52 @@
+import {
+	NetworkParams,
+	buildAccountList,
+	AccountListChoice,
+	fetchCheckEligibility,
+	getInput,
+} from '../utils/';
+
+export default async function checkEligibility(networkParams: NetworkParams): Promise<void> {
+	const lskAddress = await getInput({ message: 'Your LSK Address' });
+
+	const result = await fetchCheckEligibility(lskAddress, networkParams);
+	if (!result.account && result.multisigAccounts.length === 0) {
+		console.log(`No Eligible Claim for Address: ${lskAddress}.`);
+		return process.exit(1);
+	}
+
+	const accountList = await buildAccountList(result, networkParams);
+
+	const accountGroupedByClaimStatus = accountList.reduce(
+		(
+			acc: {
+				claimed: AccountListChoice[];
+				unclaimed: AccountListChoice[];
+			},
+			account,
+		) => {
+			if (account.claimed) {
+				acc.claimed.push(account);
+			} else {
+				acc.unclaimed.push(account);
+			}
+			return acc;
```

Comment:
Optional but may be we can write this as:
```suggestion
			account.claimed ? acc.claimed.push(account) : acc.unclaimed.push(account);
			return acc;
```